src/WPF/WPF.Viewer/Samples/Data/ManageFeatures/ManageFeatures.xaml.cs

```diff
@@ -0,0 +1,443 @@
+                await serviceGeodatabase.LoadAsync();
+
+                // Gets the feature table from the service geodatabase, referring to the Damage Assessment feature service.
+                // Creating the feature table from the feature service will cause the service geodatabase to be null.
+                _damageFeatureTable = serviceGeodatabase.GetTable(0);
+
+                //// UPDATE ATTRIBUTES - When the table loads, use it to discover the domain of the typdamage field.
+                _damageFeatureTable.Loaded += DamageTable_Loaded;
+
+                // Create a feature layer to visualize the features in the table.
+                _damageLayer = new FeatureLayer(_damageFeatureTable);
+
+                // Add the layer to the map.
+                MyMapView.Map.OperationalLayers.Add(_damageLayer);
+
+                // Zoom to the United States.
+                _ = MyMapView.SetViewpointCenterAsync(new MapPoint(-10800000, 4500000, SpatialReferences.WebMercator), 3e7);
+
+                // Bind the list of method names to the ComboBox.
+                MethodPicker.ItemsSource = _methodList;
+            }
+            catch (Exception ex)
+            {
+                MessageBox.Show(ex.Message, ex.GetType().Name, MessageBoxButton.OK, MessageBoxImage.Error);
+            }
+        }
+
+        private void DamageTable_Loaded(object sender, EventArgs e)
+        {
+            // This code needs to work with the UI, so it needs to run on the UI thread.
+            Dispatcher.Invoke(() =>
+            {
+                // Get the relevant field from the table.
+                ServiceFeatureTable table = (ServiceFeatureTable)sender;
+                Field typeDamageField = table.Fields.First(field => field.Name == AttributeFieldName);
+
+                // Get the domain for the field.
+                CodedValueDomain attributeDomain = (CodedValueDomain)typeDamageField.Domain;
+
+                // Update the ComboBox with the attribute values.
+                DamageTypeDropDown.ItemsSource = attributeDomain.CodedValues.Select(codedValue => codedValue.Name);
+            });
+        }
+
+        private async void OnComboBoxSelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+            // Clear all potentially hooked GeoViewTapped events.
+            MyMapView.GeoViewTapped -= MapView_Tapped_CreateFeature;
+            MyMapView.GeoViewTapped -= MapView_Tapped_DeleteFeature;
+            MyMapView.GeoViewTapped -= MapView_Tapped_UpdateAttribute;
+            MyMapView.GeoViewTapped -= MapView_Tapped_UpdateGeometry;
+
+            // Hold a variable for the ComboBox's selected value.
+            ComboBox comboBox = (ComboBox)sender;
+
+            if (comboBox.SelectedIndex == 0)
+            {
+                // Update the label.
+                InstructionLabel.Content = "Tap to add a feature.";
```

Comment:
Maybe `Tap on the GeoView to add a new feature.`

Reply:
I think `map` might be a little more intuitive than `GeoView`